08.ls_object/printer.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Printer
+  require_relative 'file_detail'
+
+  NUMBER_OF_COLUMN = 3
+
+  def initialize(directory, long_format, files)
+    @long_format = long_format
+    @files_detail = files_detail_array(directory, files)
+  end
+
+  def files_detail_array(directory, files)
+    files.map { |name| FileDetail.new(directory, name) }
+  end
+
+  def print
+    @long_format ? detail_format : short_format
```

Comment:
```suggestion
    @long_format ? long_format : short_format
```

shortの対義語を考えるとlong_formatの方が良さそうです。

ただ、これだと`def initialize(directory, long_format, files)`のlong_formatと名前が被るのが気持ち悪いですね。

以下の記事を参考にして修正方法を考えてみてください。

[Rubyではメソッド名と同じ名前のローカル変数を宣言しないようにしよう \- Qiita](https://qiita.com/jnchito/items/fb438aa0786bb3cbf919)